invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/elements/icon.overrides

```diff
@@ -3,7 +3,8 @@
 ***********************************************/
 
 .inline-id-icon {
-  height: @fontSizeBase;
+  height: 1rem;
```

Comment:
would it make sense to rename the class to `open-id-branding` (and all it's usages) as we discussed in the chat?